File: samples/Infralution.Localization.Wpf/SampleApp_VB/LocalizedEnumConverter.vb
#Region "File Header"
'
' COPYRIGHT:   Copyright 2009 
'              Infralution
'
#End Region

'' <summary>
'' Defines a type converter for translating enum values defined in this project
'' </summary>
Public Class LocalizedEnumConverter
    Inherits Infralution.Localization.Wpf.ResourceEnumConverter

    '' <summary>
    '' Create a new instance of the converter using translations from the given resource manager
    '' </summary>
    '' <param name="type"></param>
    Public Sub New(ByVal enumType As Type)
        MyBase.New(enumType, My.Resources.ResourceManager)
    End Sub

End Class

File: samples/Infralution.Localization.Wpf/SampleApp_CS/LocalizedEnumConverter.cs
#region File Header
//
// COPYRIGHT:   Copyright 2009 
//              Infralution
//
#endregion
using System;
using System.Collections.Generic;
using System.Text;
namespace WpfApp
{
    /// <summary>
    /// Defines a type converter for translating enum values defined in this project
    /// </summary>
    class LocalizedEnumConverter : Infralution.Localization.Wpf.ResourceEnumConverter
    {
        /// <summary>
        /// Create a new instance of the converter using translations from the given resource manager
        /// </summary>
        /// <param name="type"></param>
        public LocalizedEnumConverter(Type type)
            : base(type, Properties.Resources.ResourceManager)
        {
        }
    }
}
